ee-security/charts/helm.yaml

```diff
@@ -0,0 +1,6 @@
+build:
+  uri: https://github.com/wildfly/quickstart.git
+  contextDir: ee-security
```

Comment:
minor thing but to keep it in sync - all other helm.yaml files in this PR keep the same order:
```
uri
ref
contextDir
```